tools/llm_bench/llm_bench_utils/model_utils.py

```diff
@@ -181,18 +181,26 @@ def analyze_args(args):
             model_args['config'] = config
     if model_framework == 'ov':
         set_default_param_for_ov_config(model_args['config'])
+        if 'ATTENTION_BACKEND' not in model_args['config'] and use_case in ['text_gen', 'vlm'] and args.device != "NPU":
+            model_args['config']['ATTENTION_BACKEND'] = PA_ATTENTION_BACKEND
+        if model_args['config'].get('ATTENTION_BACKEND', '') == PA_ATTENTION_BACKEND and args.device == "NPU":
```

Comment:
I think, this is unnecessary check,  general rule is that the user is always right about what he wants to use. Just let pipeline decide how to handle this request - fail with mistake or switch back to sdpa(in case if npu surprisingly  starts supporting it, you will not need to update llm_bench)

Reply:
removed